### ISSUE TITLE
Log search highlight color hard to discern on Windows10/Default color theme

### ISSUE THREAD

**Issue by wlhlm** [MultiMC/MultiMC5#2476]
System Information
-----------------------------
MultiMC version: 0.6.2-develop-1266

Operating System: Windows 10 1803 x64

MultiMC theme: Icons: Default, Colors: System

Summary of the issue or suggestion:
----------------------------------------------

Searching through logs highlights text in a light grey on a white background, which makes it hard to discern.

What should happen:
------------------------------

Search highlight color should be easier to spot in log view. There is always the classic white text on blue background, but I'm not married to a particular color, just something more legible.

Logs/Screenshots:
----------------------------
![untitled](https://user-images.githubusercontent.com/1928113/48977695-81073680-f09f-11e8-9668-6d35f2166614.png)

Challenge: spot the text hightlight ;)

**Comment by tehbasshunter** [comment]
There actually is this setting in MMC though I'm not sure particular log part would fall under it.

**Comment by wlhlm** [comment]
@tehbasshunter Which setting are you referring to?

**Comment by tehbasshunter** [comment]
After being given a screenshot I was actually thinking of the wrong thing.

**Comment by peterix** [comment]
The highlight color is (should be) taken from the system ... what does a highlighted instance icon look like on your system?

**Comment by peterix** [comment]
And how do those look like in Windows Explorer?

**Comment by wlhlm** [comment]
This shows MultiMC and Explorer next to each other. For both screenshots I have the InfernalMobs instance highlighted in MultiMC, and `MulticMC.exe` highlighted in Explorer. The difference between the screenshots is, that in the first one I have MultiMC focused and in the second one I have Explorer focused.

MultiMC focused:

![mmcfocus](https://user-images.githubusercontent.com/1928113/49035994-c407fd80-f1b6-11e8-94d3-1c528c735d34.png)

Explorer focused:

![explorerfocus](https://user-images.githubusercontent.com/1928113/49036003-cb2f0b80-f1b6-11e8-8e34-7571c2c2e6a4.png)

Just to iterate, I have set the MMC Icon theme to "Default" and Color theme to "System". Not sure if this is affected by some Windows 10 theme settings. As far as I can tell, I've enabled Transparency effects and set default app mode to "Dark" (looking at the menu Personalize -> Color) and allowed Windows to pick a custom accent color from my background.

**Comment by wlhlm** [comment]
Yes, Dark/Bright use a blue highlight color are definitely more usable.

Maybe this is fixed by upgrading QT (though I'm completely talking out of my ass here)...

**Comment by peterix** [comment]
Maybe, but that's not gonna happen until I drop XP support entirely.

**Comment by RonRicoSuave** [comment]
I can verify the same thing happens with custom colors.   Also, the problem only shows up with the search button in the log.  Highlighting a line manually shows up as expected. 

Search button clicked:
![multimc_search](https://user-images.githubusercontent.com/2335114/53999120-e25f8c80-4107-11e9-9a6b-b8cfe823f5e9.jpg)

Line highlighted manually:
![multimc_highlight](https://user-images.githubusercontent.com/2335114/53999130-ec818b00-4107-11e9-8c2f-075265497bd0.jpg)

**Comment by RonRicoSuave** [comment]
Just for a little more info I have another piece of software using QT 5.4.1.0 that predates this one and it doesn't have any type of issue with highlights anywhere in the GUI, so I doubt a newer version than yours would solve this.  Judging by the highlight around the outside of the multi-line text control I'm seeing in the manually selected version above I'm thinking it might just be an issue of that control not having focus, although the default background selection color is still a medium gray on my system.
